code/modules/projectiles/sentries.dm

```diff
@@ -54,6 +54,11 @@
 	spark_system.set_up(5, 0, src)
 	spark_system.attach(src)
 
+	if(!CHECK_BITFIELD(gun.turret_flags, TURRET_RADIAL))
+		range = gun.turret_range
+	else
+		range = gun.turret_range - 2
```

Comment:
This should be done l48, you are setting range twice. Also this should be using trinary ( ? operator)